tests/wrappers/test_autoreset.py

```diff
@@ -62,6 +64,46 @@ def test_autoreset_reset_info():
     assert isinstance(info, dict)
 
 
+def test_make_autoreset():
```

Comment:
My initial worry was that some environments might not be guaranteed to terminate with random actions, but actually I think all environments are, since all except blackjack-v1 have max_episode_steps specified, and blackjack will terminate in less than some finite number of steps no matter what actions are taken. So I implemented the pattern so now it's applied to all first-party environments. It adds something like 30 seconds to the test battery.